plugins/uefi/fu-plugin-uefi.c

```diff
@@ -200,7 +367,14 @@ fu_plugin_update (FuPlugin *plugin,
 		g_message ("Boot Information:\n%s", boot_variables);
 	}
 
-	return TRUE;
+	/* perform the update */
+	g_debug ("Performing UEFI capsule update");
+	fu_plugin_set_status (plugin, FWUPD_STATUS_SCHEDULING);
+	if (!fu_plugin_uefi_update_splash (&error_splash)) {
+		g_warning ("failed to upload BGRT splash text: %s",
+			   error_splash->message);
+	}
+	return fu_plugin_uefi_update_resource (re, hardware_instance, blob_fw, error);
```

Comment:
So the call to `efibootmgr `above was intentionally happening last in the function.  It's not meant to debug fwupd, but rather to debug what `fwupdate` has done.

Reply:
Oops, let me remove that, sorry.